release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1166,6 +1166,14 @@ If you have Operator projects that were previously created or maintained with Op
 
 * xref:../operators/operator_sdk/helm/osdk-hybrid-helm-updating-projects.adoc#osdk-upgrading-projects_osdk-hybrid-helm-updating-projects[Updating Hybrid Helm-based Operator projects]
 
+[discrete]
+[id="ocp-4-11-cluster-operators-naming"]
+==== Cluster Operators no longer referred to as platform Operators
+
+{product-title} documentation previously referred to cluster Operators interchangeably with the alternative naming "platform Operators". Because this double naming could cause confusion about the type of Operators being discussed, the term "platform Operator" is no longer used when referring to cluster Operators, which are represented by `ClusterOperator` API objects. The documentation sets for {product-title} 4.11 and earlier have been updated to now solely use the term "cluster Operator".
```

Comment:
Oh, one thing actually, should it use {product-version} instead of 4.11?

Reply:
Looks like we have 2x as many `4.11` as `{product-version}` in this file. I suppose it doesn't matter as much since this file is only in the 4.11 branch, but I will update it!